client/src/components/tools.jsx

```diff
@@ -32,36 +33,36 @@ const Tools = () => {
     <div className="gm_tools">
       <div className="gm_tools-content">
         <Menu pointing>
-          <Menu.Item
-            disabled
-            active={selected === 'account'}
-            onClick={updateTab('account')}
-          >
-            User Management
-          </Menu.Item>
+          {' '}
           <Menu.Item
             active={selected === 'online'}
             onClick={updateTab('online')}
           >
             Who's Online
           </Menu.Item>
+          <Menu.Item
+            active={selected === 'account'}
+            onClick={updateTab('account')}
+          >
+            User Management
+          </Menu.Item>
           <Menu.Item active={selected === 'items'} onClick={updateTab('items')}>
             Item Search
           </Menu.Item>
           <Menu.Item active={selected === 'chars'} onClick={updateTab('chars')}>
             Player Search
-          </Menu.Item>
+          </Menu.Item>{' '}
           <Menu.Item active={selected === 'yells'} onClick={updateTab('yells')}>
             Yells
           </Menu.Item>
         </Menu>
-        {/* {selected === 'account' && <Accounts />} */}
         {selected === 'online' && <OnlineList />}
+        {selected === 'account' && <Accounts />}
```

Comment:
Keep it commented out for now, as we don't want users to use it yet.

Reply:
As with my prior comment, Accounts will be where people register as it is coded as a shared page with login.